src/main/java/seedu/address/model/LoanRecords.java

```diff
@@ -22,6 +22,15 @@ public LoanRecords() {
 
     }
 
+    /**
+     * Populate sample loans into the system.
+     */
+    public void populateLoans () {
```

Comment:
This method is no longer required for testing since the others will be working on their respective SampleDataUtil soon so let's take it out?